lab-raziyeh/server.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+//node modules
+const http = require('http');
+const url = require('url');
+const queryString = require('querystring');
+
+//npm modules
+const cowsay = require('cowsay');
+
+//app modules
+const parseBody = require('./lib/parse-body.js');
+
+//module constants
+const PORT = process.env.PORT || 3000;
+
+
+//module logic
+const server = http.createServer(function(req, res) {
+  req.url = url.parse(req.url);
+  req.url.query = queryString.parse(req.url.query);
+
+//Method : GET
+  if (req.method === 'GET') {
+    if(req.url.pathname === '/') {
+      res.writeHead( 200, { 'Content-Type': 'text/plain'});
+      res.write('Hello World');
+      res.end();
+    }
+    if (req.url.pathname === '/cowsay'){
+      if (req.url.query.text) {
+        res.writeHead(200);
+        res.write(cowsay.say({text:req.url.query.text}));
+        res.end();
+      } else {
```

Comment:
@Raziyehbazargan If using a return, there are alternate options to using else statements - try this out on your next major conditional based piece of functionality - it takes a bit of getting used to but really helps with code cleanliness and debugging.